client/app/bundles/course/group/pages/GroupShow/CategoryCard.jsx

```diff
@@ -133,7 +133,7 @@ const CategoryCard = ({
         onClick: handleEdit,
       });
     }
-    if (canManageGroups) {
+    if (canManageCategory) {
```

Comment:
previously, the permission for canManageGroups also allow students (who are the manager, previously made possible) to see the page containing members of the groups. But now, since students are no longer able to become a manager, and also no longer able to see the page, then basically this permission setting is no longer required.

I will remove all the canManageGroups related stuffs from here

Reply:
Have you double checked the logic of `canManageCategory` and `canManageGroups` which are derived from the backend side?